src/Nethermind/Nethermind.Core/Account.cs

```diff
@@ -100,5 +100,44 @@ public Account WithChangedCodeHash(Hash256 newCodeHash)
         {
             return new(newCodeHash, this);
         }
+
+        public AccountStruct ToStruct() => new(Nonce, Balance, StorageRoot, CodeHash);
+    }
+
+    public readonly struct AccountStruct
```

Comment:
This is a 128 bytes struct (0.125 kB); want to make sure are passing by `in` etc everywhere otherwise big copies (if low GC)

Reply:
How this comment ended? I see that in lot of places this is returned, no `TryGet` or ref semantics there? I'm asking as I'm applying these changes to Paprika and it looks that we're potentially copying to much here?